src/api.ts

```diff
@@ -349,6 +349,10 @@ export interface ResultSet {
      */
     columns: Array<string>;
 
+    /** Types of columns.
+     */
+    columnTypes: Array<string>;
```

Comment:
Would love some description here for the valid values for the strings in this array. From the tests here, it looks like empty string is a valid value, which could be surprising.

If the list of supported values is static, this should also be declared as a union of actual string value options. e.g. `Array<"BLOB" | "TEXT" | ...>`

Reply:
I documented the reason for empty strings.

The question on valid values is more complicated. SQLite is pretty relaxed in the types it allows as long as they're within the rules of type affinity: https://www.sqlite.org/datatype3.html#determination_of_column_affinity

Reply:
It looks like the valid values are INTEGER, TEXT, BLOB, REAL, and NUMERIC.  Can there be other values?  If no, then the type is pretty clearly a union of only these string values, right?